### PR TITLE
W: [fr] fr-Client-Troubleshooting.md

### PR DESCRIPTION
**Pull request by gegeweb**

### CODE REVIEW

**Comment by ann0see** [comment]
Is this PR for the new translation?

**Comment by gegeweb** [comment]
Yes, I've updated the Jamulus Explorer url as in the EN version in translate branch.
So, ready for review to be merged in translate.

**Comment by ann0see** [comment]
Ok. Can you assign trebmuh for this pr? He should merge this if everything is ok.

**Comment by gegeweb** [comment]
> Ok. Can you assign trebmuh for this pr? He should merge this if everything is ok.

I think I can't (or I don't find how to do that).

**Comment by ann0see** [comment]
Ok. Done that. So you can't use the assign button on the right?

**Comment by gegeweb** [comment]
> Ok. Done that. So you can't use the assign button on the right?

No, I can't. Disabled for me.

**Comment by trebmuh** [comment]
Pretty good translation @gegeweb .
I've seen you using `&nbsp;` in other pages you've worked on. Did you forget to do that here? or is it on purpose?

Other than that, it's ready to be merged from my point of view.

**Comment by gegeweb** [comment]
> I've seen you using `&nbsp;` in other pages you've worked on. Did you forget to do that here? or is it on purpose?

`&nbsp;` is for HTML. On Markdown (I'm on macOS) I use `ALT+MAJ+SPACE` (`U+00A0` UTF-8 character) : Jekkyl convert these with `&nbsp;` for the HTML output.

File: wiki/fr/fr-Client-Troubleshooting.md [hunk 1]
```diff
@@ -1,75 +1,77 @@
...
 
-Dans certains cas, il se peut que ce soit votre fournisseur d'accès Internet qui bloque votre utilisation pour Jamulus. Voir la note sur la [page de dépannage du serveur](Server-Troubleshooting#personne-ne-peut-se-connecter-à-mon-serveur-mais-je-peux-my-connecter-localement) à propos de « Personne ne peut se connecter à mon serveur ».
+Dans certains cas, il se peut que ce soit votre FAI qui bloque votre utilisation pour Jamulus. Voir la note sur la [page de dépannage du serveur](Server-Troubleshooting#personne-ne-peut-se-connecter-à-mon-serveur-mais-je-peux-my-connecter-localement) à propos de « Personne ne peut se connecter à mon serveur ».
 
-### Des problèmes pour utiliser Garageband (ou un autre logiciel de MAO) avec Jamulus ? 
+### Des problèmes pour utiliser Garageband (ou autre STAN) avec Jamulus ?
```
Comment (gegeweb) [inline wiki/fr/fr-Client-Troubleshooting.md:61]: I suggest `<abbr title="Station de travail audio numérique">STAN</abbr>` here.

File: wiki/fr/fr-Client-Troubleshooting.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+Start with the simple stuff: make sure your instrument/microphone and headphones are connected to the correct sockets. Make sure no other applications like your browser, video conferencing tool etc. is also using your soundcard. You should shut those down when using Jamulus. If that all looks OK and the problem persists, it's likely there's an issue with your sound device settings. This will depend on your particular setup (platform, hardware, software and drivers), so it's best to ask on [the forums](https://sourceforge.net/p/llcon/discussion/software/) for guidance.
+
```
Comment (gegeweb) [inline wiki/fr/fr-Client-Troubleshooting.md]: English paragraph to be deleted here. ;) I'll fix this evening (when going back at home)

File: wiki/fr/fr-Client-Troubleshooting.md (hunk 1 above)
Comment (trebmuh) [reply wiki/fr/fr-Client-Troubleshooting.md:61]: As usual, not before the English "DAW" got a abbr in the original English text too. Let's discuss that for after the 3.6.2 release.

File: wiki/fr/fr-Client-Troubleshooting.md (hunk 2 above)
Comment (trebmuh) [reply wiki/fr/fr-Client-Troubleshooting.md]: Well spotted. I've removed it now.

**Comment by trebmuh** [comment]
Thanks for the precision about nbsp.
I've fixed the English leftover text now.
And I've answered to the abbr comment.

Since you didn't mention any other issue, I'm merging it to translates now.

**Comment by gegeweb** [comment]
> Since you didn't mention any other issue, I'm merging it to translates now.

OK, thanks.

**Pull request merged**